src/components/site/linearProgress/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const LinearProgress = ({ min, max, value, id, labelledById }) => (
```

Comment:
close implementation of the existing `react-md` component using ARIA roles on `div` elements